README.md

```diff
@@ -1,291 +1,234 @@
 
-👤 **Author2**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
-
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+-   **User keys - login**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- CONTRIBUTING -->
-
 ## 🤝 Contributing <a name="contributing"></a>
 
-Contributions, issues, and feature requests are welcome!
+> Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+> Feel free to check the [issues page](https://github.com/gdvl777/Metrics/issues).
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- SUPPORT -->
-
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
-
-If you like this project...
+> If you like this project show support by following this account
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
+> -   Microverse for providing the opportunity to learn Git and GitHub in a collaborative environment.
 
-## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+> -   GitHub Docs for providing a wealth of information on Git and GitHub.
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
-
-- **[Question_2]**
-
-  - [Answer_2]
+> - Creative Commons for providing the design of this work.
+> - You can click on the link to see the original design:
+> - [original design on Behance](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). 
+> - Nelson Sakwa in Behance, for her original design idea.
+> - You can click on the link to see the Nelson Sakwa profile on Behance.
+> - [Nelson Sakwa profile on Behance](https://www.behance.net/sakwadesignstudio)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+> This project is [MIT](MIT.md).
```

Comment:
??? You are in the incorrect branch

Reply:
> * [x]  Kindly  I think it would be better to provide a  valid **LICENSE** link in your **README** as open source software needs to be licensed to provide rights to **modify**, **distribute** other than creator 😂
>   _we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._

- [ ] Hello, as mentioned here by the previous reviewer, have a look at the link to your project license. it is currently broken. Kindly have it fixed

```suggestion
> This project is [MIT](./MIT.md).
```

![Screenshot from 2023-10-14 23-10-08](https://github.com/gdvl777/metrics/assets/112261237/f40f8718-e3c8-4921-87fb-f6bfe582a266)

Reply:
- [ ] **OPTIONAL** I also keep getting the same error, please make sure to correct this! 🤓 🔧